README.md

```diff
@@ -3,26 +3,34 @@
 
 # BossShopPro
```

Comment:
I think this should be Genesis aswell, right?

Reply:
Yeah. Thanks for catching that.